### PR TITLE
Reworks the effects of brain damage

### PR DESCRIPTION
**Pull request by BiancaWilkson**
## What Does This PR Do

This PR replaces the forced messages on high brain damage with a variety of effects, scaling based on how much brain damage you have. Tweaks some of the other brain damage messages/effects as well.
## Why It's Good For The Game

Brain damage currently is in a weird spot, where it either has very little effect on gameplay outside of forcing your character to say annoying OOC things, right up until you drop dead. This causes a few issues, most notably that if someone doesn't care about roleplaying, there's not a ton to motivate you to actually fix said brain damage. The other issue is that brain damage is a really good assassination method, since until you hit the "say funny things" threshold, there's no real way to know you're dying.

I tried to stick to somewhat realistic brain damage symptoms, but suggestions for more effects are always accepted.

Lastly, because IPC posibrains are MMIs, and not brains, none of these symptoms work for them. If needed, I can add a separate proc for MMIs to have similar effects, since being in a damaged posibrain/MMI is currently the worst fate known to man anyway.
## Testing

I drank a lot of mercury and waited around to make sure the effects worked
## Changelog
:cl:
tweak: Reworked the effects of brain damage
/:cl:

### CODE REVIEW

File: code/modules/surgery/organs/brain.dm [hunk 1]
```diff
@@ -125,6 +125,60 @@
...
+	if(prob(5))
+		owner.Weaken(15 SECONDS)
+		owner.Jitter(15 SECONDS)
+		to_chat(owner, "<span class='warning'>You start to have a seizure.</span>")
+	else if(prob(4))
+		owner.Silence(45 SECONDS)
+		to_chat(owner, "<span class='warning'>You open your mouth to speak, but no sound comes out.</span>")
+	else if(prob(5))
+		owner.EyeBlind(30 SECONDS)
+		to_chat(owner, "<span class='warning'>Your vision gives out.</span>")4
```
Comment (lewcc) [inline code/modules/surgery/organs/brain.dm:180]: 4
```suggestion
		to_chat(owner, "<span class='warning'>Your vision gives out.</span>")
```

**Review by lewcc — COMMENTED** [review]
4

File: code/modules/surgery/organs/brain.dm (hunk 1 above)
Comment (BiancaWilkson) [reply code/modules/surgery/organs/brain.dm]: 4got to push

**Comment by Mitchs98** [comment]
I like the rest of the PR. But I don't honestly see why all the brain damage lines have to be removed. I personally find them quite funny. Part of them could maybe be trimmed/tweaked, but flat removal? I don't see why that's needed for the rest of the changes in the PR to exist.

**Review by DGamerL — COMMENTED** [review]
IM A CHOO CHOO

For the rest, very interesting idea that I like, I'm just not entirely sure on every brain damage thing being on something like a `prob(5)` times 10

File: code/modules/surgery/organs/brain.dm [hunk 2]
```diff
@@ -125,6 +125,65 @@
...
+/obj/item/organ/internal/brain/proc/handle_critical_brain_damage()
+	if(prob(4))
+		owner.Silence(45 SECONDS)
+		to_chat(owner, "<span class='warning'>You open your mouth to speak, but no sound comes out.</span>")
+	else if(prob(5))
+		owner.EyeBlind(30 SECONDS)
+		to_chat(owner, "<span class='warning'>Your vision gives out.</span>")
+	else if(prob(5))
+		owner.Weaken(10 SECONDS)
+		owner.Jitter(10 SECONDS)
```
Comment (lewcc) [inline code/modules/surgery/organs/brain.dm:184]: I think you might want to increase the jitter here like how epilepsy does it; it's based on time. You can probably set it back to zero on a timer.

Comment (BiancaWilkson) [reply code/modules/surgery/organs/brain.dm:184]: Epilepsy intends on being essentially a permanent jitter, that's why it's number is so high. Though it does feel somewhat fitting for critical brain damage to make you permanently jittery, as well.

Comment (lewcc) [reply code/modules/surgery/organs/brain.dm]: Yeah, what I mean is that jitter's intensity is based on the remaining time of the effect. It's silly, but it's how it's currently done. If you want to make them "seize", you're best off setting the jitter stupid high, and then resetting it on a timer.

File: code/modules/surgery/organs/brain.dm [hunk 3]
```diff
@@ -125,6 +125,65 @@
...
+		owner.vomit()
+		to_chat(owner, "<span class='warning>'You feel nauseous.</span>")
+
+/obj/item/organ/internal/brain/proc/handle_moderate_brain_damage()
+	if(prob(4))
+		owner.Confused(20 SECONDS)
+		to_chat(owner, "<span class='warning'>It's suddenly difficult to walk straight.</span>")
+	else if(prob(5))
+		owner.EyeBlurry(15 SECONDS)
+		to_chat(owner, "<span class='warning'>Your vision unfocuses.")
```
Comment (lewcc) [inline code/modules/surgery/organs/brain.dm:155]:
```suggestion
		to_chat(owner, "<span class='warning'>Your vision unfocuses.</span>")
```

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**